public/js/components/EditorSearchBar.js

```diff
@@ -91,23 +99,41 @@ const EditorSearchBar = React.createClass({
     this.search(query);
   },
 
-  onKeyUp(e) {
+  traverseResultsPrev(shortcut, e) {
```

Comment:
nit: `traverseResultsPrev` is a little wordy, perhaps `prevResult` or `findPrev`?